README.md

```diff
@@ -9,59 +7,70 @@ This is a Couchbase connector node module for [Loopback](http://loopback.io/) wi
 ### Install
 
 ```
-npm install loopback-connector-couchbase3 --save
+npm install loopback-connector-couchbaseX --save
 ```
 
-### Config
+### DataSource Config in LoopBack
```

Comment:
The only difference is cb5 the username/password are not managed per bucket, could you drop a note here to make the reader easier to find the diff and understand what's changed(best to link to an official CB doc).